### PR TITLE
Maintenance update

### PR DESCRIPTION
**Pull request by tim77**

### CODE REVIEW

File: ua.org.brezblock.q4wine.yml [hunk 1]
```diff
@@ -32,6 +32,13 @@ finish-args:
 cleanup:
   - '*.a'
   - '*.la'
+  - /include
+  - /lib/*.a
```
Comment (Erick555) [inline ua.org.brezblock.q4wine.yml]: there is `*.a` and `*.la` above so those aren't needed.

File: ua.org.brezblock.q4wine.yml [hunk 2]
```diff
@@ -32,6 +32,13 @@ finish-args:
...
+  - /lib/*.la
+  - /lib/cmake
+  - /lib/pkgconfig
+  - /lib32/*.a
```
Comment (Erick555) [inline ua.org.brezblock.q4wine.yml]: same as `/lib/*.a`

File: ua.org.brezblock.q4wine.yml [hunk 3]
```diff
@@ -32,6 +32,13 @@ finish-args:
 cleanup:
   - '*.a'
   - '*.la'
+  - /include
+  - /lib/*.a
+  - /lib/*.la
+  - /lib/cmake
```
Comment (Erick555) [inline ua.org.brezblock.q4wine.yml:36]: you may clean cmake & pkgconfig for /lib32 as well

Comment (tim77) [reply ua.org.brezblock.q4wine.yml:36]: Great! TY. I wish flatpak will have one day predefined ultimate preset for cleanup thing.

**Pull request merged**